_posts/ja/posts/2017-11-11-release-0.15.1.md

```diff
@@ -80,7 +80,7 @@ GUIの設定は`-resetguisettings`引数が使われた際、設定が消去さ
 
 詳細については[リリースノート][]を参照ください。ダウンロードは[ダウンロードページ][]からできます。
 
-ご不明な点がありましたら、[IRC][]もしくは[Slack][]のチャットルームまでご連絡ください。
+ご不明な点がありましたら、[IRC][]もしくはのチャットルームまでご連絡ください。
```

Comment:
This doesn't seem to be quite right.  Google Translate shows it as "If you have any questions, please contact IRC or a chat room."  However, it seems to be a fairly minor problem and an old announcement, so I think we can ignore it.